### PR TITLE
chore: changed scrape interval to 60s and batch size to 10000

### PR DESCRIPTION
**Pull request by ankitnayan**
* Changed hostmetrics scrape interval from 30s to 60s to reduce samples and improve perf
* Changed apm metrics scrape interval from 30s to 60s to reduce samples and improve perf
* Changed batch size of metrics exporter to 10K from 1K

### CODE REVIEW

File: deploy/docker-swarm/clickhouse-setup/otel-collector-config.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ receivers:
       network:
 processors:
   batch:
-    send_batch_size: 1000
+    send_batch_size: 10000
```
Comment (prashant-shahi) [inline deploy/docker-swarm/clickhouse-setup/otel-collector-config.yaml:25]:
```
processors:
  batch:
    timeout: 10s
    send_batch_size: 10000
    send_batch_max_size: 11000
```

`send_batch_max_size` is the upper limit of the batch size which ensures that larger batches are split into smaller units.

Ref:
https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor/batchprocessor#readme

File: deploy/docker-swarm/clickhouse-setup/otel-collector-metrics-config.yaml [hunk 2]
```diff
@@ -9,12 +9,12 @@ receivers:
     config:
       scrape_configs:
         - job_name: "otel-collector"
-          scrape_interval: 30s
+          scrape_interval: 60s
```
Comment (prashant-shahi) [inline deploy/docker-swarm/clickhouse-setup/otel-collector-metrics-config.yaml:12]: Looks good to me, but with new metrics builder the graph plot would be slightly different than before.

Related to this: https://github.com/SigNoz/engineering-pod/issues/487

File: deploy/docker-swarm/clickhouse-setup/otel-collector-metrics-config.yaml [hunk 3]
```diff
@@ -9,12 +9,12 @@ receivers:
...
           static_configs:
             - targets: ["otel-collector:8889"]
 processors:
   batch:
-    send_batch_size: 1000
+    send_batch_size: 10000
```
Comment (prashant-shahi) [inline deploy/docker-swarm/clickhouse-setup/otel-collector-metrics-config.yaml:17]: same as that of OtelCollector config

File: deploy/docker/clickhouse-setup/otel-collector-metrics-config.yaml [hunk 4]
```diff
@@ -9,12 +9,12 @@ receivers:
...
       scrape_configs:
         - job_name: "otel-collector"
-          scrape_interval: 30s
+          scrape_interval: 60s
           static_configs:
             - targets: ["otel-collector:8889"]
 processors:
   batch:
-    send_batch_size: 1000
+    send_batch_size: 10000
```
Comment (prashant-shahi) [inline deploy/docker/clickhouse-setup/otel-collector-metrics-config.yaml:17]: same as that of OtelCollector config

File: deploy/docker/clickhouse-setup/otel-collector-config.yaml [hunk 5]
```diff
@@ -22,7 +22,7 @@ receivers:
       network:
 processors:
   batch:
-    send_batch_size: 1000
+    send_batch_size: 10000
```
Comment (prashant-shahi) [inline deploy/docker/clickhouse-setup/otel-collector-config.yaml:25]: same as that of OtelCollector config

**Review by prashant-shahi — CHANGES_REQUESTED** [review]
@ankitnayan left some comments. please check

File: deploy/docker-swarm/clickhouse-setup/otel-collector-config.yaml (hunk 1 above)
Comment (ankitnayan) [reply deploy/docker-swarm/clickhouse-setup/otel-collector-config.yaml:25]: Will `send_batch_max_size` be used if we already specify `send_batch_size`?

File: deploy/docker-swarm/clickhouse-setup/otel-collector-metrics-config.yaml (hunk 2 above)
Comment (ankitnayan) [reply deploy/docker-swarm/clickhouse-setup/otel-collector-metrics-config.yaml:12]: Yeah, I agree. The above issue should be useful later.

File: deploy/docker-swarm/clickhouse-setup/otel-collector-config.yaml (hunk 1 above)
Comment (prashant-shahi) [reply deploy/docker-swarm/clickhouse-setup/otel-collector-config.yaml:25]: yes, it is.
`send_batch_size` would be the lower limit, while `send_batch_max_size` would be the upper limit.

File: deploy/docker-swarm/clickhouse-setup/otel-collector-metrics-config.yaml (hunk 3 above)
Comment (ankitnayan) [reply deploy/docker-swarm/clickhouse-setup/otel-collector-metrics-config.yaml:17]: added

File: deploy/docker-swarm/clickhouse-setup/otel-collector-config.yaml (hunk 1 above)
Comment (ankitnayan) [reply deploy/docker-swarm/clickhouse-setup/otel-collector-config.yaml:25]: added

**Pull request merged**